README.md

```diff
@@ -1,23 +1,33 @@
 # CXDocs
 Compound X rules displayed as web pages. Written in Python 2.7, served with Flask and bootstraped with bootswatch.
 
-to set up, 
+#set up
+##Python module instalation
+###On Ubuntu 16.04 
 1) $git clone https://github.com/bleehu/CXDocs.git
 2) $sudo pip install flask 
 3) $sudo pip install flask_sqlalchemy
 4) $sudo apt-get install python-psycopg2
 
+
+
 to test, use:
-$python start.py
+`$python start.py`
 then in a web browser, go to localhost:5000/levelup
 
 For LAN, use:
-$python start.py -i local.ip.goes.here
-to find your local ip address on linux, use ifconfig. 
-to find your local ip address on windows, use ipconfig in bash
+`$python start.py -i local.ip.goes.here`
+to find your local ip address on linux, use ifconfig in a terminal. 
```

Comment:
Confirm "ifconfig" as opposed to "ipconfig"?

Reply:
confirmed.